Makefile

```diff
@@ -45,3 +45,10 @@ ifeq ($(OS),Windows_NT)
 else
 		@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-30s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 endif
+
+.PHONY: update-modules
```

Comment:
Can you add some help text (similar to the other commands) so that this ends up in `make help`?